apps/openmw/mwscript/controlextensions.cpp

```diff
@@ -167,7 +168,11 @@ namespace MWScript
                 virtual void execute (Interpreter::Runtime& runtime)
                 {
                     MWWorld::Ptr ptr = MWBase::Environment::get().getWorld ()->getPlayerPtr();
-                    runtime.push (ptr.getClass().getCreatureStats(ptr).getStance(MWMechanics::CreatureStats::Stance_Run));
+                    const MWWorld::Class &cls = ptr.getClass();
+
+                    bool isMoving = std::abs(cls.getMovementSettings(ptr).mPosition[0]) > .5 || std::abs(cls.getMovementSettings(ptr).mPosition[1]) > .5;
```

Comment:
I took this check from CharacterController::update. If > 0 is better, I will update PR.